recipe/bld.bat

```diff
@@ -2,15 +2,34 @@
 set CC=cl.exe
 set CXX=cl.exe
 
-mkdir build
-cd build
-cmake ^
+REM The pkg-config path and executable are needed to find the freeglut header and library
+set PKG_CONFIG_PATH="%LIBRARY_BIN%\pkgconfig;%LIBRARY_LIB%\pkgconfig;%LIBRARY_PREFIX%\share\pkgconfig;%BUILD_PREFIX%\Library\lib\pkgconfig;%BUILD_PREFIX%\Library\bin\pkgconfig"
+set PKG_CONFIG_EXECUTABLE=%LIBRARY_BIN%\pkg-config
+
+REM The default %LIBRARY_LIB%\freeglut.lib can't be found by CMake even is DGLUT_glut_LIBRARY_RELEASE is defined.
+REM While linking C executable src\app\jiv.exe two files are required glut.lib AND(!) freeglutd.lib instead of freeglut.lib. 
+REM So we rename freeglut.lib to glut.lib and freeglutd.lib.
+REM At the same time pkg-config is looking for glut.pc but not freeglut.pc. Renaming it fixes the issue.
+copy %LIBRARY_LIB%\pkgconfig\freeglut.pc %LIBRARY_LIB%\pkgconfig\glut.pc
+copy %LIBRARY_LIB%\freeglut.lib %LIBRARY_LIB%\glut.lib
+copy %LIBRARY_LIB%\freeglut.lib %LIBRARY_LIB%\freeglutd.lib
```

Comment:
I thought `d` was for "debug"... CMake will look for both `freeglut.lib` and `freeglutd.lib` and will set `GLUT_glut_LIBRARY` to the right one based on the current configuration (either release or debug).

See https://github.com/Kitware/CMake/blob/v3.26.4/Modules/FindGLUT.cmake#L83-L98.

So this tells me that we are potentially building in debug mode...

I feel worried that we have to rename things here.

Reply:
How will it be better to resolve the issue without renaming files? Because the jiv's C linking step expects `glut.lib` but not `freeglut.lib`. `freeglut` was built in release mode https://github.com/AnacondaRecipes/freeglut-feedstock/blob/f6a0e0fd055eeeca1994fefc2657fa4c6455f665/recipe/bld.bat#L8. Or do you mean that `jasper` is building in **debug** mode if CMake finds `freeglutd.lib`?